### DISCUSSION TITLE
Import template into existing app

### DISCUSSION THREAD

**Discussion by techthrasher** [Budibase/budibase#12778] [Help] [answered]
Hi!  I'm building an app and realized one of the awesome templates (business directory) could be really helpful as a *component* of my existing app.  However, for the life of me I can't figure out if it's possible to import that template into my existing app.  I suspect I could probably do it piecemeal or using cut and paste, but was hoping I could use a "one-click" method to avoid that process.  Is it possible?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @techthrasher,

Currently you can't copy components between screens nor different apps at the moment. Your best bet is to either work off the template app, or create each component to match when comparing to the template.

Heres a link to the relevant [enhancement](https://github.com/Budibase/budibase/issues/7136) that you are describing in this discussion.

I hope this makes sense.

**Reply by techthrasher** [reply]
Thank you @ConorWebb96.  That's definitely functionality that would be valuable, but I can understand how it might be a tough hill to climb for the Budibase devs.
